UnityProject/Assets/Scripts/Items/Weapons/WeaponNetworkActions.cs

```diff
@@ -84,6 +84,10 @@ public void SetNewDamageValues(float newAttackSpeed, float newAttackDamage, Dama
 	public void ServerPerformMeleeAttack(GameObject victim, Vector2 attackDirection, BodyPartType damageZone, LayerType layerType)
 	{
 		if (victim == null) return;
+		if (playerMove.ObjectIsBuckling.OrNull()?.gameObject != null && playerMove.ObjectIsBuckling is MovementSynchronisation)
```

Comment:
This does mean you can't destroy chairs that people are tied to

Reply:
nope, `MovementSync` is mobs only
so you'll hit chairs still